src/components/follow.tsx

```diff
@@ -37,6 +44,79 @@ function getStyles({ theme }: Format): SerializedStyles {
 	return styles(kicker, inverted);
 }
 
+const FollowIcon: FC<IconProps> = ({ isFollowing }) => {
+	return (
+		<>
+			{isFollowing && (
```

Comment:
This might be simpler as:

```tsx
if (isFollowing) {
  return <svg className="following-icon"...>...</svg>
} else {
  return <svg className="follow-icon"...>...</svg>
}
```

But perhaps best not to touch it until merging #1355 to prevent conflicts.